pkg/agent/flowexporter/exporter/exporter.go

```diff
@@ -178,20 +193,25 @@ func (exp *flowExporter) initFlowExporter(collector net.Addr) error {
 }
 
 func (exp *flowExporter) sendFlowRecords() error {
-	sendAndUpdateFlowRecord := func(key flowexporter.ConnectionKey, record flowexporter.FlowRecord) error {
-		dataSet := ipfix.NewSet(ipfixentities.Data, exp.templateID, false)
-		if err := exp.sendDataSet(dataSet, record, exp.templateID); err != nil {
+	dataSet := ipfix.NewSet(ipfixentities.Data, exp.templateID, false)
```

Comment:
This can be moved outside to enable a unit test for this function.
If no time, please add a TODO.